pubspec.yaml

```diff
@@ -100,7 +100,7 @@ dependencies:
   desktop_webview_window: ^0.2.3
 
   # Desktop
-  window_manager: ^0.4.2
+  window_manager: ^0.4.3
```

Comment:
I noticed there's a new version of window_manager available so I'm updating it on this PR as well.